### DISCUSSION TITLE
Release does not include main at least after some time?

### DISCUSSION THREAD

**Discussion by sthibaul** [rust-lang/libc#4024] [Q&A]
Hello,

Looking at the latest release 0.2.162, it still does not include e.g. e2fd4d3349e489cf4c20330fe70b3b5004a832db , which was committed in june. How can that happen? Aren't releases taking commits from main eventually after some time?

Samuel

**Comment by sthibaul** [comment]
Ok, I see that we need an explicit cherry-pick into 0.2

**Reply by redwan-islam-rumman** [reply]
Good
